### PR TITLE
use consistent (c) for copyright symbol

### PR DESCRIPTION
**Pull request by SteveL-MSFT**
Looks like vscode also removed some trailing whitespace

Fix https://github.com/PowerShell/PowerShell/issues/4917

### CODE REVIEW

**Review by PaulHigin — APPROVED** [review]
These are a lot of changes to look at, but it all looks good as far as I can tell.

**Review by adityapatwardhan — APPROVED** [review]
LGTM

**Comment by adityapatwardhan** [comment]
https://github.com/PowerShell/PowerShell/pull/5210/files?w=1 can be use to hide the whitespace only changes.

**Review by markekraus — APPROVED** [review]
LGTM

File: src/System.Management.Automation/engine/DscResourceInfo.cs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 //-----------------------------------------------------------------------
 // <copyright company="Microsoft">
-//    Copyright (C) 2013 Microsoft Corporation
+//    Copyright (c) 2013 Microsoft Corporation
```
Comment (mirichmo) [inline src/System.Management.Automation/engine/DscResourceInfo.cs]: There are a bunch of entries that include the date. It should be removed and made consistent with the others shouldn't it? This comment applies to a bunch of other changes if it needs to be addressed.

Copyright (c) Microsoft Corporation, All rights reserved.

**Review by mirichmo — CHANGES_REQUESTED** [review]

File: src/System.Management.Automation/engine/DscResourceInfo.cs (hunk 1 above)
Comment (SteveL-MSFT) [reply src/System.Management.Automation/engine/DscResourceInfo.cs]: Yeah, my understanding is that the date doesn't matter.  Will update.

**Review by mirichmo — APPROVED** [review]

File: test/powershell/Modules/Microsoft.PowerShell.Core/PSSessionConfiguration.Tests.ps1 [hunk 2]
```diff
@@ -656,7 +656,7 @@ namespace PowershellTestConfigNamespace
                     SessionType = 'Default'
                     Author = 'User'
                     CompanyName = 'Microsoft Corporation'
-                    Copyright = 'Copyright (c) 2011 Microsoft Corporation. All rights reserved.'
+                    Copyright (c) Microsoft Corporation. All rights reserved.'
```
Comment (iSazonov) [inline test/powershell/Modules/Microsoft.PowerShell.Core/PSSessionConfiguration.Tests.ps1]: Typo. Please revert "Copyright = ".

File: src/Microsoft.Management.Infrastructure.CimCmdlets/AssemblyInfo.cs [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /*============================================================================
- * Copyright (C) Microsoft Corporation, All rights reserved.
+ * Copyright (c) Microsoft Corporation, All rights reserved.
```
Comment (iSazonov) [inline src/Microsoft.Management.Infrastructure.CimCmdlets/AssemblyInfo.cs]: Please replace "," with dot.
I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."

We have many such patterns below.

File: src/Microsoft.Management.Infrastructure.CimCmdlets/RemoveCimSessionCommand.cs [hunk 4]
```diff
@@ -1,5 +1,5 @@
 //
-//    Copyright (C) Microsoft.  All rights reserved.
+//    Copyright (c) Microsoft.  All rights reserved.
```
Comment (iSazonov) [inline src/Microsoft.Management.Infrastructure.CimCmdlets/RemoveCimSessionCommand.cs]: Extra spaces.
I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."

File: src/Microsoft.PowerShell.Commands.Diagnostics/PdhHelper.cs [hunk 5]
```diff
@@ -1,5 +1,5 @@
 //
-//    Copyright (C) Microsoft.  All rights reserved.
+//    Copyright (c) Microsoft.  All rights reserved.
```
Comment (iSazonov) [inline src/Microsoft.PowerShell.Commands.Diagnostics/PdhHelper.cs]: Extra spaces.
I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."

File: src/Microsoft.PowerShell.Commands.Diagnostics/PdhSafeHandle.cs [hunk 6]
```diff
@@ -1,5 +1,5 @@
 //
-//    Copyright (C) Microsoft.  All rights reserved.
+//    Copyright (c) Microsoft.  All rights reserved.
```
Comment (iSazonov) [inline src/Microsoft.PowerShell.Commands.Diagnostics/PdhSafeHandle.cs]: Extra spaces.
I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."

File: src/Microsoft.PowerShell.Commands.Diagnostics/resources/GetEventResources.txt [hunk 7]
```diff
@@ -1,4 +1,4 @@
-#Copyright (c) 2008 Microsoft Corporation
+#Copyright (c) Microsoft Corporation
```
Comment (iSazonov) [inline src/Microsoft.PowerShell.Commands.Diagnostics/resources/GetEventResources.txt]: I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."

File: src/Microsoft.PowerShell.Commands.Management/commands/management/ControlPanelItemCommand.cs [hunk 8]
```diff
@@ -1,5 +1,5 @@
 //
-//    Copyright (C) Microsoft.  All rights reserved.
+//    Copyright (c) Microsoft.  All rights reserved.
```
Comment (iSazonov) [inline src/Microsoft.PowerShell.Commands.Management/commands/management/ControlPanelItemCommand.cs]: I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."

File: src/Microsoft.PowerShell.Commands.Management/commands/management/InvokeWMIMethodCommand.cs [hunk 9]
```diff
@@ -1,5 +1,5 @@
 //
-//    Copyright (C) Microsoft.  All rights reserved.
+//    Copyright (c) Microsoft.  All rights reserved.
```
Comment (iSazonov) [inline src/Microsoft.PowerShell.Commands.Management/commands/management/InvokeWMIMethodCommand.cs]: Extra spaces.
I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."

File: src/Microsoft.PowerShell.Commands.Management/commands/management/RegisterWMIEventCommand.cs [hunk 10]
```diff
@@ -1,5 +1,5 @@
 //
-//    Copyright (C) Microsoft.  All rights reserved.
+//    Copyright (c) Microsoft.  All rights reserved.
```
Comment (iSazonov) [inline src/Microsoft.PowerShell.Commands.Management/commands/management/RegisterWMIEventCommand.cs]: The same.

I see below many the same patterns.

File: src/Microsoft.PowerShell.Commands.Utility/commands/utility/ShowCommand/ShowCommand.cs [hunk 11]
```diff
@@ -1,6 +1,6 @@
 ﻿//-----------------------------------------------------------------------
 // <copyright company="Microsoft">
-//    Copyright © Microsoft Corporation.  All rights reserved.
+//    Copyright (c) Microsoft Corporation.  All rights reserved.
```
Comment (iSazonov) [inline src/Microsoft.PowerShell.Commands.Utility/commands/utility/ShowCommand/ShowCommand.cs]: Extra spaces.
I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."

Below I see many the same patterns.

File: src/Microsoft.WSMan.Management/resources/WsManResources.txt [hunk 12]
```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2008 Microsoft Corporation
+# Copyright (c) Microsoft Corporation
```
Comment (iSazonov) [inline src/Microsoft.WSMan.Management/resources/WsManResources.txt]: I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."

File: src/Modules/Windows-Core+Full/CimCmdlets/CimCmdlets.psd1 [hunk 13]
```diff
@@ -2,7 +2,7 @@
 GUID="{Fb6cc51d-c096-4b38-b78d-0fed6277096a}"
 Author="Microsoft Corporation"
 CompanyName="Microsoft Corporation"
-Copyright="© Microsoft Corporation. All rights reserved."
+Copyright="(c) Microsoft Corporation. All rights reserved."
```
Comment (iSazonov) [inline src/Modules/Windows-Core+Full/CimCmdlets/CimCmdlets.psd1]: I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."

File: src/Modules/Windows-Full/Microsoft.PowerShell.ODataUtils/Microsoft.PowerShell.ODataUtils.psd1 [hunk 14]
```diff
@@ -22,7 +22,7 @@ Author = 'Microsoft Corporation'
 CompanyName = 'Microsoft Corporation'
 
 # Copyright statement for this module
-Copyright = '(c) 2014 Microsoft. All rights reserved.'
+Copyright (c) Microsoft. All rights reserved.'
```
Comment (iSazonov) [inline src/Modules/Windows-Full/Microsoft.PowerShell.ODataUtils/Microsoft.PowerShell.ODataUtils.psd1]: Typo. Please revert "Copyright =".

I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."

File: src/System.Management.Automation/engine/DscResourceInfo.cs [hunk 15]
```diff
@@ -1,6 +1,6 @@
 //-----------------------------------------------------------------------
 // <copyright company="Microsoft">
-//    Copyright (C) 2013 Microsoft Corporation
+//    Copyright (c) Microsoft Corporation
```
Comment (iSazonov) [inline src/System.Management.Automation/engine/DscResourceInfo.cs]: I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."

File: src/System.Management.Automation/engine/lang/interface/PSParseError.cs [hunk 16]
```diff
@@ -1,5 +1,5 @@
 /********************************************************************++
-    Copyright (C) Microsoft Corporation, 2003
+    Copyright (c) Microsoft Corporation, 2003
```
Comment (iSazonov) [inline src/System.Management.Automation/engine/lang/interface/PSParseError.cs]: I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."

File: src/System.Management.Automation/engine/remoting/fanin/WSManPlugin.cs [hunk 17]
```diff
@@ -1,7 +1,7 @@
 // ----------------------------------------------------------------------
 //
 //  Microsoft Windows NT
-//  Copyright (C) Microsoft Corporation, 2007.
+//  Copyright (c) Microsoft Corporation, 2007.
```
Comment (iSazonov) [inline src/System.Management.Automation/engine/remoting/fanin/WSManPlugin.cs]: I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."

Below too.

File: src/powershell-native/nativemsh/pwrshcommon/ClrHostWrapper.h [hunk 18]
```diff
@@ -1,7 +1,7 @@
 // ----------------------------------------------------------------------
 //
 //  Microsoft Windows NT
-//  Copyright (C) Microsoft Corporation, 2014.
+//  Copyright (c) Microsoft Corporation, 2014.
```
Comment (iSazonov) [inline src/powershell-native/nativemsh/pwrshcommon/ClrHostWrapper.h]: I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."

File: src/powershell-native/nativemsh/pwrshcommon/NativeMsh.h [hunk 19]
```diff
@@ -1,7 +1,7 @@
 // ----------------------------------------------------------------------
 //
 //  Microsoft Windows NT
-//  Copyright (C) Microsoft Corporation, 2005.
+//  Copyright (c) Microsoft Corporation, 2005.
```
Comment (iSazonov) [inline src/powershell-native/nativemsh/pwrshcommon/NativeMsh.h]: I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."

File: src/powershell-native/nativemsh/pwrshexe/MshResources.rc [hunk 20]
```diff
@@ -1,4 +1,4 @@
-// Copyright (C) 2005 Microsoft Corporation
+// Copyright (c) Microsoft Corporation
```
Comment (iSazonov) [inline src/powershell-native/nativemsh/pwrshexe/MshResources.rc]: I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."

File: src/powershell-native/nativemsh/pwrshplugin/pwrshheaders.h [hunk 21]
```diff
@@ -1,7 +1,7 @@
 // ----------------------------------------------------------------------
 //
 //  Microsoft Windows NT
-//  Copyright (C) Microsoft Corporation, 2008.
+//  Copyright (c) Microsoft Corporation, 2008.
```
Comment (iSazonov) [inline src/powershell-native/nativemsh/pwrshplugin/pwrshheaders.h]: I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."

File: test/powershell/Modules/Microsoft.PowerShell.Security/TestData/CatalogTestData/UserConfigProv/UserConfigProv.psd1 [hunk 22]
```diff
@@ -2,7 +2,7 @@
     ModuleVersion = '3.0.0.1'
     Author = 'fayzas'
     CompanyName = 'Microsoft Corporation'
-    Copyright = '(c) 2013 Microsoft Corporation. All rights reserved.'
+    Copyright (c) Microsoft Corporation. All rights reserved.'
```
Comment (iSazonov) [inline test/powershell/Modules/Microsoft.PowerShell.Security/TestData/CatalogTestData/UserConfigProv/UserConfigProv.psd1]: Typo. Please revert "Copyright = "

File: test/tools/Modules/HelpersCommon/HelpersCommon.psd1 [hunk 23]
```diff
@@ -12,7 +12,7 @@ GUID = 'cc1c8e94-51d1-4bc1-b508-62bc09f02f54'
 
 CompanyName = 'Microsoft Corporation'
 
-Copyright = 'Copyright (C) Microsoft Corporation, All rights reserved.'
+Copyright = 'Copyright (c) Microsoft Corporation, All rights reserved.'
```
Comment (iSazonov) [inline test/tools/Modules/HelpersCommon/HelpersCommon.psd1]: Please replace "," with dot.
I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."

File: tools/packaging/packaging.psd1 [hunk 24]
```diff
@@ -2,7 +2,7 @@
 GUID="41857994-4283-4757-a932-0b0edb104913"
 Author="Microsoft Corporation"
 CompanyName="Microsoft Corporation"
-Copyright="© Microsoft Corporation. All rights reserved."
+Copyright="(c) Microsoft Corporation. All rights reserved."
```
Comment (iSazonov) [inline tools/packaging/packaging.psd1]: I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."

File: tools/packaging/project/powershell.nuspec [hunk 25]
```diff
@@ -11,7 +11,7 @@
     <projectUrl>https://github.com/powershell/powershell</projectUrl>
     <iconUrl>https://github.com/PowerShell/PowerShell/blob/master/assets/Powershell_64.png</iconUrl>
     <description>This package contains the PowerShell Core for $runtime$.</description>
-    <copyright>Copyright © Microsoft Corporation</copyright>
+    <copyright>Copyright (c) Microsoft Corporation</copyright>
```
Comment (iSazonov) [inline tools/packaging/project/powershell.nuspec]: I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."

File: test/powershell/Modules/Microsoft.PowerShell.Core/PSSessionConfiguration.Tests.ps1 (hunk 2 above)
Comment (SteveL-MSFT) [reply test/powershell/Modules/Microsoft.PowerShell.Core/PSSessionConfiguration.Tests.ps1]: Will fix

File: src/Microsoft.Management.Infrastructure.CimCmdlets/AssemblyInfo.cs (hunk 3 above)
Comment (SteveL-MSFT) [reply src/Microsoft.Management.Infrastructure.CimCmdlets/AssemblyInfo.cs]: I'll fix all the occurrences

**Comment by SteveL-MSFT** [comment]
@iSazonov I think I got all of them.

File: src/System.Management.Automation/engine/InternalCommands.cs [hunk 26]
```diff
@@ -295,12 +295,12 @@ protected override void ProcessRecord()
                                 else
                                 {
                                     // we write null out because:
-                                    // PS C:\> �$null | ForEach-object {$_.aa} | ForEach-Object {$_ + 3}�
+                                    // PS C:\> �$null | ForEach-object {$_.aa} | ForEach-Object {$_ + 3}�
```
Comment (iSazonov) [inline src/System.Management.Automation/engine/InternalCommands.cs]: Ah, @SteveL-MSFT  please review the whole file - unicode chars?

File: src/System.Management.Automation/help/CommandHelpProvider.cs [hunk 27]
```diff
@@ -585,7 +585,7 @@ private string FindHelpFile(CmdletInfo cmdletInfo)
                 // e.g., <AssemblyName>.ni.dll as supposed to <AssemblyName>.dll.
 
                 // When cmdlet metadata is generated for the 'HelpFile' field, we use the name assembly and we append '-Help.xml' to it.
-                // Because of this, if the cmdlet is part of an N�gen assembly, then 'HelpFile' field will be pointing to a help file which does not exist.
+                // Because of this, if the cmdlet is part of an N�gen assembly, then 'HelpFile' field will be pointing to a help file which does not exist.
```
Comment (iSazonov) [inline src/System.Management.Automation/help/CommandHelpProvider.cs]: Unicode char?

File: src/System.Management.Automation/help/CommandHelpProvider.cs [hunk 28]
```diff
@@ -604,7 +604,7 @@ private string FindHelpFile(CmdletInfo cmdletInfo)
                     if (String.IsNullOrEmpty(location))
                     {
                         // If the help file could not be found, then it is possible that the actual assembly name is something like
-                        // <Name>.ni.dll, e.g., MyAssembly.ni.dll, so let�s try to find the original help file in the cmdlet metadata.
+                        // <Name>.ni.dll, e.g., MyAssembly.ni.dll, so let�s try to find the original help file in the cmdlet metadata.
```
Comment (iSazonov) [inline src/System.Management.Automation/help/CommandHelpProvider.cs]: Unicode char?

File: src/System.Management.Automation/engine/remoting/commands/NewPSSessionOptionCommand.cs [hunk 29]
```diff
@@ -226,31 +226,31 @@ public AuthenticationMechanism ProxyAuthentication
...
 
         /// <summary>
-        /// The duration for which PowerShell should wait before it times out on cancel operations 
-        /// (close runspace or stop powershell). For instance, when the user hits ctrl-C, 
-        /// New-PSSession cmdlet tries to call a stop on all remote runspaces which are in the Opening state. 
-        /// The user wouldn�t mind waiting for 15 seconds, but this should be time bound and of a shorter duration. 
+        /// The duration for which PowerShell should wait before it times out on cancel operations
+        /// (close runspace or stop powershell). For instance, when the user hits ctrl-C,
+        /// New-PSSession cmdlet tries to call a stop on all remote runspaces which are in the Opening state.
+        /// The user wouldn�t mind waiting for 15 seconds, but this should be time bound and of a shorter duration.
```
Comment (iSazonov) [inline src/System.Management.Automation/engine/remoting/commands/NewPSSessionOptionCommand.cs]: Unicode char?

File: src/System.Management.Automation/engine/remoting/commands/NewPSSessionOptionCommand.cs [hunk 30]
```diff
@@ -386,11 +386,11 @@ public int OpenTimeout
...
 
         /// <summary>
-        /// The duration for which PowerShell should wait (in milliseconds) before it 
+        /// The duration for which PowerShell should wait (in milliseconds) before it
         /// times out on cancel operations (close runspace or stop powershell). For
         /// instance, when the user hits ctrl-C, New-PSSession cmdlet tries to call a
-        /// stop on all remote runspaces which are in the Opening state. The user 
-        /// wouldn�t mind waiting for 15 seconds, but this should be time bound and of a 
+        /// stop on all remote runspaces which are in the Opening state. The user
+        /// wouldn�t mind waiting for 15 seconds, but this should be time bound and of a
```
Comment (iSazonov) [inline src/System.Management.Automation/engine/remoting/commands/NewPSSessionOptionCommand.cs]: Unicode char?

**Review by iSazonov — COMMENTED** [review]
Please replace "All Rights Reserved" with "All rights reserved".

Oh, so many unrelated formatting changes! - I'm afraid I missed something.

File: src/Microsoft.Management.Infrastructure.CimCmdlets/RemoveCimSessionCommand.cs [hunk 31]
```diff
@@ -1,5 +1,5 @@
 //
-//    Copyright (C) Microsoft.  All rights reserved.
+//    Copyright (c) Microsoft Corporation. All Rights Reserved.
```
[the remark this answers is not in the document]
Comment (SteveL-MSFT) [reply src/Microsoft.Management.Infrastructure.CimCmdlets/RemoveCimSessionCommand.cs]: I'll replace them all

File: src/System.Management.Automation/engine/InternalCommands.cs (hunk 26 above)
Comment (SteveL-MSFT) [reply src/System.Management.Automation/engine/InternalCommands.cs]: Will fix and replace the 'fancy quote' with a 'regular quote' character

**Comment by SteveL-MSFT** [comment]
Well, this turned out to be a much bigger set of changes than originally anticipated, but @iSazonov I appreciate your attention to detail.  I think I got them all and reviewed them with https://github.com/PowerShell/PowerShell/pull/5210/files?w=1

File: src/Modules/Unix/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psd1 [hunk 32]
```diff
@@ -2,7 +2,7 @@
 GUID="1DA87E53-152B-403E-98DC-74D7B4D63D59"
 Author="Microsoft Corporation"
 CompanyName="Microsoft Corporation"
-Copyright="© Microsoft Corporation. All rights reserved."
+Copyright="(c) Microsoft Corporation. All rights reserved."
```
Comment (iSazonov) [inline src/Modules/Unix/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psd1]: Skipped "Copyright"
`Copyright="Copyright (c) Microsoft Corporation. All rights reserved."`

File: src/Modules/Windows-Core+Full/PSDiagnostics/PSDiagnostics.psd1 [hunk 33]
```diff
@@ -2,7 +2,7 @@
     GUID="c61d6278-02a3-4618-ae37-a524d40a7f44 "
     Author="Microsoft Corporation"
     CompanyName="Microsoft Corporation"
-    Copyright="© Microsoft Corporation. All rights reserved."
+    Copyright="(c) Microsoft Corporation. All rights reserved."
```
Comment (iSazonov) [inline src/Modules/Windows-Core+Full/PSDiagnostics/PSDiagnostics.psd1]: Skipped "Copyright"
`Copyright="Copyright (c) Microsoft Corporation. All rights reserved."`

File: src/Modules/Windows-Core/Microsoft.PowerShell.Diagnostics/Microsoft.PowerShell.Diagnostics.psd1 [hunk 34]
```diff
@@ -2,7 +2,7 @@
 GUID="CA046F10-CA64-4740-8FF9-2565DBA61A4F"
 Author="Microsoft Corporation"
 CompanyName="Microsoft Corporation"
-Copyright="© Microsoft Corporation. All rights reserved."
+Copyright="(c) Microsoft Corporation. All rights reserved."
```
Comment (iSazonov) [inline src/Modules/Windows-Core/Microsoft.PowerShell.Diagnostics/Microsoft.PowerShell.Diagnostics.psd1]: Skipped "Copyright"
`Copyright="Copyright (c) Microsoft Corporation. All rights reserved."`

File: src/Modules/Windows-Core/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psd1 [hunk 35]
```diff
@@ -2,7 +2,7 @@
 GUID="1DA87E53-152B-403E-98DC-74D7B4D63D59"
 Author="Microsoft Corporation"
 CompanyName="Microsoft Corporation"
-Copyright="© Microsoft Corporation. All rights reserved."
+Copyright="(c) Microsoft Corporation. All rights reserved."
```
Comment (iSazonov) [inline src/Modules/Windows-Core/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psd1]: Skipped "Copyright"
`Copyright="Copyright (c) Microsoft Corporation. All rights reserved."`

File: src/Modules/Windows-Full/Microsoft.PowerShell.LocalAccounts/Microsoft.PowerShell.LocalAccounts.psd1 [hunk 36]
```diff
@@ -4,7 +4,7 @@ ModuleVersion = '1.0.0.0'
 GUID = '8e362604-2c0b-448f-a414-a6a690a644e2'
 Author = 'Microsoft Corporation'
 CompanyName = 'Microsoft Corporation'
-Copyright = '© Microsoft Corporation. All rights reserved.'
+Copyright = '(c) Microsoft Corporation. All rights reserved.'
```
Comment (iSazonov) [inline src/Modules/Windows-Full/Microsoft.PowerShell.LocalAccounts/Microsoft.PowerShell.LocalAccounts.psd1]: Skipped "Copyright"
`Copyright="Copyright (c) Microsoft Corporation. All rights reserved."`

**Review by iSazonov — APPROVED** [review]
LGTM but I think we need someone else to check line by line because there's changes in code lines in cs files.

**Comment by iSazonov** [comment]
@SteveL-MSFT Many thanks that now we have nice copyright strings!

Please clarify (it seems I already asked):
1. We have Microsoft Copyright in most files and MIT for the Repo - is there a conflict? CoreFX has MIT copyright in all files and Repo. We have both  Microsoft Copyright and MIT in LICENSE.txt and MIT in README.md.
2. What copyright should we add in new files or shouldn't?

**Comment by SteveL-MSFT** [comment]
@iSazonov 

1. let me follow-up on that, but I don't think the copyright notice is really necessary
2. my understanding is that we really don't need to add it to new files

**Pull request merged**